scripts/mctl/firmwares.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "uuid": "f215f65a-ef5a-4af1-a035-7b890d0958f5",
+    "vendor": "supermicro",
+    "model": [
+      "x11sse-f"
+    ],
+    "filename": "X11SSE0.B25",
+    "version": "2.5",
+    "component": "bios",
+    "checksum": "c6da4c4638c1d94c27dbea1206434cef",
```

Comment:
This was probably my mistake in the first place, can you add the missing checksum prefix to this and the others below 🙏